### PR TITLE
Update the HLT customisation

### PR DESCRIPTION
**Pull request by fwyzard**
Load a skeleton CUDAService when running on the CPU.
Make the label for the quadruplets consistent on CPU and GPU.
Enable offloading ECAL unpacking and multifit.

### CODE REVIEW

File: HLTrigger/Configuration/python/customizeHLTforPatatrack.py [hunk 1]
```diff
@@ -52,12 +66,20 @@ def customise_cpu_pixel(process):
...
+    process.hltPixelTracksHitQuadruplets = _caHitNtupletCUDA.clone(
         idealConditions = False,
         pixelRecHitSrc = "hltSiPixelRecHitSoA",
         onGPU = False
     )
 
+    process.hltPixelTracksSoA = cms.EDAlias(
+        hltPixelTracksHitQuadruplets = cms.VPSet(
+            cms.PSet(
+                type = cms.string("32768TrackSoATHeterogeneousSoA")
```
Comment (makortel) [inline HLTrigger/Configuration/python/customizeHLTforPatatrack.py:78]: I'm wondering `32768` here. Is the C++ type `HeterogeneousSoA<pixelTrack::TrackSoA>`? `edmToFriendlyClassName "HeterogeneousSoA<pixelTrack::TrackSoA>"` gives `pixelTrackTrackSoAHeterogeneousSoA`.

Comment (fwyzard) [reply HLTrigger/Configuration/python/customizeHLTforPatatrack.py:78]:
```python
process.ascii = cms.OutputModule("AsciiOutputModule",
    verbosity = cms.untracked.uint32(2),
    outputCommands = cms.untracked.vstring(
        'drop *',
        'keep *_hltPixelTracksHitQuadruplets_*_*'
    )
)
```
gives
```
Branch Type = Event
Process Name = HLTGRun
ModuleLabel = hltPixelTracksHitQuadruplets
Branch ID = 1538655069
Class Name = HeterogeneousSoA<TrackSoAT<32768> >
Friendly Class Name = 32768TrackSoATHeterogeneousSoA
Product Instance Name =
```

which agrees with
```bash
$ edmToFriendlyClassName 'HeterogeneousSoA<TrackSoAT<32768>>'
32768TrackSoATHeterogeneousSoA
```

I guess `edmToFriendlyClassName` clearly cannot know about template parameters with default values ?

**Comment by fwyzard** [comment]
@amassiro does the ECAL part look good to you (pre-#457) ?

File: HLTrigger/Configuration/python/customizeHLTforPatatrack.py (hunk 1 above)
Comment (makortel) [reply HLTrigger/Configuration/python/customizeHLTforPatatrack.py:78]: Thanks, I didn't look carefully enough then. The `edmToFriendlyClassName` does only text manipulation, so if the C++ type is wrong, so is the output friendly name :)

**Comment by amassiro** [comment]
> @amassiro does the ECAL part look good to you (pre-#457) ?

yes, ECAL modules seem ok

**Comment by fwyzard** [comment]
Thanks.

**Pull request merged**